pcdsdevices/spectrometer.py

```diff
@@ -318,3 +318,34 @@ class TMOSpectrometer(BaseInterface, GroupDevice):
     yag_y = Cpt(BeckhoffAxisNoOffset, ':MMS:07', kind='normal')
     yag_z = Cpt(BeckhoffAxisNoOffset, ':MMS:08', kind='normal')
     yag_theta = Cpt(BeckhoffAxisNoOffset, ':MMS:09', kind='normal')
+
+
+class HXRSpectrometer(BaseInterface, GroupDevice):
+    """
+    HXR Single Shot Spectrometer motion components class.
+
+    Parameters:
+    -----------
+    prefix : str
+        Base PV for spectrometer motors.
+
+    name : str
+        Alias for the device.
+    """
+
+    tab_component_names = True
+
+    krst = Cpt(IMS, ':441:MOTR', kind='normal',
+               doc='crystal y')
+    ksta = Cpt(IMS, ':442:MOTR', kind='normal'
```

Comment:
Missing comma here, otherwise looks good to me
```suggestion
    ksta = Cpt(IMS, ':442:MOTR', kind='normal',
```
From the automated tests:
```
pcdsdevices/spectrometer.py:341:16: E999 SyntaxError: invalid syntax
```